vaadin-accordion-flow-parent/vaadin-accordion-flow/pom.xml

```diff
@@ -44,6 +44,12 @@
             <artifactId>slf4j-simple</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>jakarta.platform</groupId>
+            <artifactId>jakarta.jakartaee-web-api</artifactId>
+            <version>10.0.0</version>
```

Comment:
Wanna add this versioning (dependencyManagement) to the root pom like done in flow to ease migrating later to 11?

Reply:
Done